subsys/bluetooth/mesh/pb_gatt_srv.c

```diff
@@ -244,9 +244,12 @@ static size_t gatt_prov_adv_create(struct bt_data prov_sd[2])
 
 dev_name:
 	if (IS_ENABLED(CONFIG_BT_MESH_PB_GATT_USE_DEVICE_NAME)) {
+		uint8_t name[BT_GAP_DEVICE_NAME_MAX_SIZE];
+		size_t name_size = bt_gap_get_device_name(name, sizeof(name));
```

Comment:
`-ENOMEM` should be checked and\or converted to zero.

Could it be static assertion instead of size checking?

Reply:
I added a check, I don't think that using assertion would be right here.